.eslintrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   rules: {
     '@typescript-eslint/array-type': 'error',
     '@typescript-eslint/explicit-module-boundary-types': 'error',
-    '@typescript-eslint/no-explicit-any': 'error',
```

Comment:
@DaimDN Why this is changed from error to warn ?
We should avoid making such common framework changes which will impact all